### PR TITLE
change

### PR DESCRIPTION
**Pull request by NAYAKSCODE**
i update a print statement on line no 16

### CODE REVIEW

File: src/test/java/crm/com/sample1/DummyTest.java [hunk 1]
```diff
@@ -13,5 +13,6 @@ public void morning()
 	public void night()
 	{
 		System.out.println(" good night india");
+		System.out.println("jay maa saraswati");
```
Comment (tarinipradhan) [inline src/test/java/crm/com/sample1/DummyTest.java:16]: please add maa @ end of saraswati

**Review by tarinipradhan — COMMENTED** [review]
please add

File: src/test/java/crm/com/sample1/DummyTest.java (hunk 1 above)
Comment (NAYAKSCODE) [reply src/test/java/crm/com/sample1/DummyTest.java:16]: ok i will do it .

**Comment by tarinipradhan** [comment]
i accept

**Pull request merged**